internal/network/manifests_generator.go

```diff
@@ -555,12 +555,14 @@ spec:
 
             [Service]
             Type=oneshot
-            ExecStart=-/bin/sh -c '                                                      \
-            lsblk -o NAME,TRAN,MOUNTPOINTS --json | jq -e \'.blockdevices[] | select(.tran == "iscsi") | select(.children) | .children[].mountpoints | select(.) | index("/sysroot") | select(.)\' > /dev/null; \
-            if [ $? = 0 ];                                                               \
-            then                                                                         \
-                echo "iSCSI boot volume detected, force network reconfiguration...";     \
-                nmcli -t -f DEVICE device status | xargs -l nmcli device reapply;        \
+            ExecStart=-/bin/sh -c ' set -x; \
+            MULTIPATH=".blockdevices[] | select(.tran == \\\"iscsi\\\") | select(.children) | .children[] | select(.children) | .children[].mountpoints | select(.) | index(\\\"/sysroot\\\")"; \
+            ISCSI_ALONE=".blockdevices[] | select(.tran == \\\"iscsi\\\") | select(.children) | .children[].mountpoints | select(.) | index(\\\"/sysroot\\\")"; \
```

Comment:
Maybe add a comment here to highlight what we are trying to do.
It's quite a hard read and could use a small explanation.